opendata.swiss/ui/app/plugins/keycloak.ts

```diff
@@ -0,0 +1,29 @@
+import Keycloak from "keycloak-js";
+import { defineNuxtPlugin } from "#app";
+
+
+export default defineNuxtPlugin((nuxtApp) => {
+  // Get config from runtimeConfig/public with fallback to defaults
+  const { keycloakUrl, keycloakRealm, keycloakClientId } = (nuxtApp.$config?.public || {}) as Record<string, string>;
+  const keycloak = new Keycloak({
+    url: keycloakUrl && typeof keycloakUrl === 'string' && keycloakUrl.length > 0 ? keycloakUrl : "https://keycloak.zazukoians.org/",
+    realm: keycloakRealm && typeof keycloakRealm === 'string' && keycloakRealm.length > 0 ? keycloakRealm : "lindas-next",
+    clientId: keycloakClientId && typeof keycloakClientId === 'string' && keycloakClientId.length > 0 ? keycloakClientId : "piveau-hub-ui",
```

Comment:
I would not hardcode defaults here. They are already set in the config and setting something falsy should result in an error anyway